### ISSUE TITLE
Protein Structure Figure

### ISSUE THREAD

**Issue by cclarkson** [malariagen/ag1000g-phase2-vgsc-report#64]
We now have this lovely looking protein structure figure, I am contacting Andrias about methods so we can integrate. The question is, where do we think it would sit best?
![model_labelled_V2](https://user-images.githubusercontent.com/13287471/73932414-2cca2a00-48d2-11ea-8ca5-332e43cb6ac4.jpeg)

**Comment by alimanfoo** [comment]
Nice. 

We'll obviously need a good figure caption to explain what's going on here. I wonder if it's worth trying to arrange a short call with Andrias so he can explain it to us in person? 

Some questions I have:
* What are the red bits?
* The solid rods are transmembrane segments? What do the colours mean?
* Where are the two pyrethroid binding sites?
* Which parts are internal to cell, which external?
* Maybe needs some way to mark more precisely where the mutations are on the sequence?

Basically we're going to need to be able to explain how this communicates the major topological features of the protein.

**Comment by cclarkson** [comment]
I'm going to PR the latest version of the figure and Andy's text so it doesn't get lost.
